css/reset-css.md

```diff
@@ -0,0 +1,146 @@
+
+### Objectives
+
+**Participants will be able to:**
+- Understand what is reset.css & why it is used.
+- Understand browser styling differences.
+
+### Specific Things To Teach
+
+- how to use reset.css
+
+
+### Materials
+
+- [Default browser styles and CSS Resets article (4 min read)](https://medium.com/@tinydinosaur/a-wordy-history-of-default-browser-styles-and-css-resets-befdd614d93b)
+- [Default browser styles and CSS Resets video (5 min watch)](https://www.youtube.com/watch?v=xLP8IM3jkgE)
+
+### Lesson
+- [Default browser styles and CSS Resets video (5 min watch)](https://www.youtube.com/watch?v=xLP8IM3jkgE)
+
+### Guided Practice
+#### Practice implementation of reset.css
+1. Create an HTML file with a linked CSS file (named style.css), a heading, a paragraph and an order list like this:
+```
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <h2>Lorem Ipsum</h2>
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+  </p>
+  <ol>
+    <li>One item</li>
+    <li>Two item</li>
+    <li>Three item</li>
+  </ol>
+</body>
+</html>
+```
+2. Create a CSS file (named style.css), and style your HTML elements like this:
+```
+h2{
+  background-color:#87CEFA;
+}
+
+p{
+  background-color:#FFB6C1;
+}
+
+ol{
+  background-color:#90EE90;
+  -moz-border-radius:5px;
+  -webkit-border-radius:5px;
+  border-radius:5px;
+}
+```
+3. Refresh your page, and then examine the webpage.
+4. You can observe that there is some white gap between each Html elements and as well as between HTML element & page sides. You can also access them by the inspect tool present in your browser.
+5. Once you examine `h2` element and `ol` element through inspect tool available in your chrome browser you can clearly observe that some styles are coming from **user agent stylesheet** like this:
+```
+user agent stylesheet
+
+h2 {
+    display: block;
+    font-size: 1.5em;
+    margin-block-start: 0.83em;
+    margin-block-end: 0.83em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-weight: bold;
+}
+```
+```
+user agent stylesheet
+
+ol {
+    display: block;
+    list-style-type: decimal;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    padding-inline-start: 40px;
+}
+```
+6. Astonish where these styles come from because in your CSS file (named style.css) you didn't mention them. These are browser specific styles which every browser provide for some HTML elements.
```

Comment:
This sentence still doesn’t make sense... how about: Are you wondering where these styles are coming from when you didn’t write them in style.css?